.drone.yml

```diff
@@ -231,10 +231,8 @@ steps:
         from_secret: github_read_token
 
   - name: test-mysql
-    image: golang:1.15
+    image: a1012112796/gitea_test_env:linux-amd64
```

Comment:
This is unrelated change for this PR

Reply:
Without this change, can't run new test code for this feature because of the low git version.